great_expectations/expectations/metrics/util.py

```diff
@@ -171,10 +171,15 @@ def get_dialect_regex_expression(column, regex, dialect, positive=True):
             dialect.dialect,
             snowflake.sqlalchemy.snowdialect.SnowflakeDialect,
         ):
-            if positive:
-                return BinaryExpression(column, literal(regex), custom_op("RLIKE"))
-            else:
-                return BinaryExpression(column, literal(regex), custom_op("NOT RLIKE"))
+            # if positive:
+            #     return BinaryExpression(column, literal(regex), custom_op("RLIKE"))
+            # else:
+            #     return BinaryExpression(column, literal(regex), custom_op("NOT RLIKE"))
```

Comment:
Do we want to remove these commented lines before merging?

Reply:
I was thinking we keep it as a hint in case snowflake makes changes so that it works as intended

Reply:
sounds good